build/github/acceptance_test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+set -x
+
+STATUS=0
+
+./privateer completion
+if [ $? -ne 0 ]; then
+    STATUS=1
+fi
+
+./privateer generate-plugin -p ./test/data/CCC.VPC_2025.01.yaml -n example
+if [ $? -ne 0 ]; then
+    STATUS=1
+fi
+
+./privateer help
+if [ $? -ne 0 ]; then
+    STATUS=1
+fi
+
+./privateer list
+if [ $? -ne 0 ]; then
+    STATUS=1
+fi
+
+./privateer run -b ./test/data/
+if [ $? -ne 0 ]; then
+    STATUS=1
+fi
+
+./privateer version
+if [ $? -ne 0 ]; then
+    STATUS=1
+fi
+
+exit $STATUS
```

Comment:
Do we want to `exit(?$)` from inside the if checks?  This means it will fail fast, and we'll know which one is failing.  Currently, we won't know which one is failing, if multiple fail. What do you think?

Downside is the developer will have to fix each failure and resubmit to see if any others fail.

I'm good with merging this and iterating.

Reply:
I think the downside you mention is the primary reason for this design. I gave it some thought and I'm happy with this approach.

Reply:
But...without some sort of messaging we will not know which command failed. Correct?

Reply:
The logs seem to be pretty verbose... you can check the CI on this pull request